config_src/drivers/nuopc_cap/mom_cap_methods.F90

```diff
@@ -271,6 +305,159 @@ subroutine mom_import(ocean_public, ocean_grid, importState, ice_ocean_boundary,
+    call state_getimport(importState, 'Faxa_dstwet', &
+        isc, iec, jsc, jec, ice_ocean_boundary%atm_fine_dust_flux(:,:), &
+        areacor=med2mod_areacor, do_sum=.true., esmf_ind=1, rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    call state_getimport(importState, 'Faxa_dstdry',  &
+        isc, iec, jsc, jec, ice_ocean_boundary%atm_fine_dust_flux(:,:), &
+        areacor=med2mod_areacor, do_sum=.true., esmf_ind=1, rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+  endif
+
+  !----
+  ! coarse dust flux from atmosphere
+  !----
+  if (associated(ice_ocean_boundary%atm_coarse_dust_flux)) then
+    ice_ocean_boundary%atm_coarse_dust_flux(:,:) = 0._ESMF_KIND_R8
+    do esmf_ind=2,4
+      call state_getimport(importState, 'Faxa_dstwet',  &
+          isc, iec, jsc, jec, ice_ocean_boundary%atm_coarse_dust_flux(:,:), &
+          areacor=med2mod_areacor, do_sum=.true., esmf_ind=esmf_ind, rc=rc)
+      if (ChkErr(rc,__LINE__,u_FILE_u)) return
+      call state_getimport(importState, 'Faxa_dstdry',  &
+          isc, iec, jsc, jec, ice_ocean_boundary%atm_coarse_dust_flux(:,:), &
+          areacor=med2mod_areacor, do_sum=.true., esmf_ind=esmf_ind, rc=rc)
+      if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    enddo
+  end if
+
+  !----
+  ! dust flux from sea ice
+  !----
+  if (associated(ice_ocean_boundary%seaice_dust_flux)) then
+    call state_getimport(importState, 'Fioi_flxdst',  &
+        isc, iec, jsc, jec, ice_ocean_boundary%seaice_dust_flux, &
+        areacor=med2mod_areacor, rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+  endif
+
+  !----
+  ! black carbon flux from atmosphere
+  !----
+  if (associated(ice_ocean_boundary%atm_bc_flux)) then
+    ice_ocean_boundary%atm_bc_flux(:,:) = 0._ESMF_KIND_R8
+    do esmf_ind=1,3
+      call state_getimport(importState, 'Faxa_bcph',  &
+          isc, iec, jsc, jec, ice_ocean_boundary%atm_bc_flux(:,:), &
+          areacor=med2mod_areacor, do_sum=.true., esmf_ind=esmf_ind, rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    enddo
+  endif
+
+  !----
+  ! black carbon flux from sea ice
+  !----
+  if (associated(ice_ocean_boundary%seaice_bc_flux)) then
+    ice_ocean_boundary%seaice_bc_flux(:,:) = 0._ESMF_KIND_R8
+    call state_getimport(importState, 'Fioi_bcpho',  &
+        isc, iec, jsc, jec, ice_ocean_boundary%seaice_bc_flux(:,:), &
+        areacor=med2mod_areacor, do_sum=.true., rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    call state_getimport(importState, 'Fioi_bcphi',  &
+        isc, iec, jsc, jec, ice_ocean_boundary%seaice_bc_flux(:,:), &
+        areacor=med2mod_areacor, do_sum=.true., rc=rc)
+    if (ChkErr(rc,__LINE__,u_FILE_u)) return
+  endif
+
+  ! Fields coming from coupler per ice category
+  if (ice_ocean_boundary%ice_ncat > 0) then
+    call state_getimport(importState, 'Sf_afracr',  &
+        isc, iec, jsc, jec, ice_ocean_boundary%afracr(:,:), rc=rc)
+    if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, &
```

Comment:
more examples of multi-line chkerr call.

Reply:
will cleanup them in the future PR